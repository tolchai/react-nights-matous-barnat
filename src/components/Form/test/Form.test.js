import React from 'react'
import { render } from 'react-testing-library'
import 'jest-styled-components'

import { Form } from '../index'

// This is just an example how to test components
describe('[components] Form', () => {
  it('should render correctly', () => {
    const renderer = render(<Form>My Form</Form>)
    expect(renderer.container).toMatchSnapshot()
  })
})
