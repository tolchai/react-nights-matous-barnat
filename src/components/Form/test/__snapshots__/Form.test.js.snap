// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[components] Form should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin: 1.4rem auto;
  max-width: 420px;
  padding: 0 1.2rem;
  width: 100%;
}

<div>
  <form
    class="c0"
  >
    My Form
  </form>
</div>
`;
